src/test/java/innohack/gem/entity/feature/TikaFeatureTests.java

```diff
@@ -60,7 +60,7 @@ private void testMetadata(TikaFeature abs) {
         } else if (metaDataCount == 1) {
           assertTrue(entry.getKey().equals("Content-Encoding"));
           if (FileUtilForTesting.isWindows(OS)) {
-            assertTrue(entry.getValue().equals("ISO-8859-1"));
+            assertTrue(entry.getValue().equals("windows-1252"));
```

Comment:
Strangely, my Windows give me `ISO-8859-1`, so my unit test always fail. Travis passes as it is Linux (and we are cehcking that it is not `windows-1252`)

Reply:
me too. im fixing this in a new PR coming soon

Reply:
err do we need to fix the formatter ah.. how come like alot of files piew the formatter..